### ISSUE TITLE
Build up Privacy pathway badges in P2PU

### ISSUE THREAD

**Issue by wiobyrne** [mozilla/webliteracymap#18]
As detailed in the WG deliverable, we should/need to build up the badges for Privacy and make them available as a proof of concept. Then, we can move on to develop pathways for the remaining competencies. 

Deliverable - http://bit.ly/BAWGprivacy

Blog post on the topic - http://bit.ly/1LQfq1K

**Comment by dajbelshaw** [comment]
Agreed, although given this is driven from a Badge Alliance working group, this will be a community effort rather than a Mozilla-resourced one (in the first instance at least). Happy to co-ordinate it here for the sake of convenience!
